packages/integrations/react/test/react-component.test.js

```diff
@@ -99,12 +99,18 @@ describe('React Components', () => {
 			const $ = cheerioLoad(html);
 			expect($('#cloned').text()).to.equal('Cloned With Props');
 		});
+
+		it('Children are parsed as React components, can be manipulated', async () => {
+			const html = await fixture.readFile('/children/index.html');
+			const $ = cheerioLoad(html);
+			expect($(".with-children-count").text()).to.equal('1');
```

Comment:
Children count would always be `1`, even with the option enabled. 

We should probably include a test where the React component has multiple children. The children count will be `1` without the experimental flag enabled, and `N` with it enabled.

Reply:
With out it enabled children is an object with no length. But I agree having multiple children is a better test so I'll update.

Reply:
Updated this test to have multiple children, better captures the intent of the test.